clang/docs/analyzer/developer-docs/Statistics.rst

```diff
@@ -0,0 +1,33 @@
+===================
+Analysis Statistics
+===================
+
+CSA enjoys two facilities to collect statistics: per translation unit and per entry point.
```

Comment:
While I share and use the CSA shorthand, I wonder if the dovs we shoul avoid using it. WDYT?

Reply:
Expanded to be on the safe side
548eaadabf32 Expand acronym